### PR TITLE
Fix issue #23

### PR DESCRIPTION
**Pull request by niclaslind**
Hi,
I did an example in Go of how to subscribe to an arbitration & respond with the signals that belongs to the arbitration frame. 

I'm think this is what you are asking for in the issue #23 , if not, maybe it can still be useful to have in the repo. 

I also updated README, so it's up to date with the new example.

### CODE REVIEW

**Comment by AleksandarFilipov** [comment]
many thanks, bouncing this back for review by another go guy @PeterWinzell

**Comment by AleksandarFilipov** [comment]
Great work. I know it's @PeterWinzell to blame for this ;) but could you please remove remove the proto files in the signalbroker-server/examples/grpc/go/timeSync/proto_files folder and refer to the "original" proto files (as done in the other examples) which are located `apps/grpc_service/proto_files`
Check my inline comment....

File: examples/grpc/go/README.md [hunk 1]
```diff
@@ -1,47 +1,24 @@
...
-go run main.go pngprint.go
-```
+	```
+	go get -u github.com/fogleman/gg
+	go get -u github.com/sirupsen/logrus
+	go get -u google.golang.org/grpc
+	```
+4. Include the generated grpc proto files in the folder proto_files, the go hook files (*.pb.go) is also generated in the folder proto_files.
+	```
+	protoc -I proto_files proto_files/*.proto  --go_out=plugins=grpc:. proto_files/*.proto
```
Comment (AleksandarFilipov) [inline examples/grpc/go/README.md]:
```suggestion
	protoc -I ../../../apps/grpc_service/proto_files ../../../apps/grpc_service/proto_files/*.proto  --go_out=plugins=grpc:. proto_files/*.proto
```

**Comment by AleksandarFilipov** [comment]
@niclaslind could you please erase examples/grpc/go/linArbitrationSub/proto_files/network_api.proto and the other *.proto so that we prevent same file in multiple folders.

**Comment by niclaslind** [comment]
Sorry, I missed that part in your first comment, I have removed the .proto files now. :)

**Comment by AleksandarFilipov** [comment]
Many thanks @niclaslind, once merged, I'll update the https://github.com/volvo-cars/signalbroker-server/#advanced-usage to reference your code and I'll close #23

**Pull request merged**